### ISSUE TITLE
[Bug]: disableIdTokenValidation config flag is undocumented and named misleadingly

### ISSUE THREAD

**Issue by luryus** [damienbod/angular-auth-oidc-client#1552]
**What Version of the library are you using?**
14.1.5

**Describe the bug**
The `disableIdTokenValidation` flag, added in #1451, is not listed in the available config options in the documentation.

This is especially problematic as the flag is badly named anyway:
- it **does not** disable all ID token validation, unlike the name would suggest
- it **does not** disable even all time validation, unlike the PR title would suggest

The flag only disables ID token expiry time validation. Iat validation is controlled by `disableIatOffsetValidation`.

**Expected behavior**
Configuration flags should be clearly named and documented to make them discoverable.

**Comment by damienbod** [comment]
fixed in version 14.2.0

Once released you can configuration it something like this:

```typescript
import { NgModule } from '@angular/core';
import { AuthModule, LogLevel } from 'angular-auth-oidc-client';

@NgModule({
  imports: [
    AuthModule.forRoot({
      config: {
        authority: '--idp--',
        redirectUrl: window.location.origin,
        postLogoutRedirectUri: window.location.origin,
        clientId: '--client_id--',
        scope: 'openid profile offline_access',
        responseType: 'code',
        silentRenew: true,
        useRefreshToken: true,
        ignoreNonceAfterRefresh: true, // this is required if the id_token is not returned
        allowUnsafeReuseRefreshToken: true, // this is required if the refresh token is not rotated
        enableIdTokenExpiredValidationInRenew: false, // required to refresh the browser if id_token is not updated after the first authentication
        autoUserInfo: false, // if the user endpoint is not supported
        logLevel: LogLevel.Debug,
      },
    }),
  ],
  exports: [AuthModule],
})
export class AuthConfigModule {}
```

**Comment by damienbod** [comment]
released v15.0.0
